lib/hooks/useOrders.ts

```
@@ -0,0 +1,47 @@
+
+export default function useOrders() {
+  const [currentTab, setCurrentTab] = useState<string>(ORDER_DISPATCH_TYPE[0]);
+
+  const { loading, error, data, refetch, networkStatus } = useContext(
+    RestaurantProvider.Context
+  );
+  const activeOrders: IOrder[] =
+    data &&
+    data?.restaurantOrders?.filter(
+      (order: IOrder) => order?.orderStatus === "PENDING"
+    );
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)